### PR TITLE
Const-corrected and updated README.md

### PR DESCRIPTION
**Pull request by AliLogic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,3 +76,8 @@ public OnDialogPerformed(playerid, dialog[], response, success)
     return 1;
 }
 ```
+
+## Credits
+Emmet_ - for easyDialog
+Southclaws - for samp-ctl
```
Comment (Southclaws) [inline README.md]: I hate being pedantic [but this is technically sampctl](https://github.com/Southclaws/sampctl/wiki/What-Does-sampctl-Mean)

**Review by Southclaws — CHANGES_REQUESTED** [review]

**Comment by AliLogic** [comment]
Users who don't have their a_samp const corrected will warnings, so if anybody has put up the const corrected default libraries, then put up a link here and i'll add it.

**Comment by AliLogic** [comment]
I've fixed my oofs (errors) and added a warning that users should use sampctl's std lib. If there's anything that I should take care of, please let me know :) Have a nice day.

**Comment by Southclaws** [comment]
Thank you!

**Review by Southclaws — APPROVED** [review]

**Pull request merged**